fe1-web/src/features/evoting/interface/Feature.ts

```diff
@@ -0,0 +1,29 @@
+import { Hash, PublicKey } from 'core/objects';
+
+export namespace EvotingFeature {
```

Comment:
I'm not sure I understand the purpose of this file. If we remove dependencies the way you did in other features that have events (like `rollCall`), will we need to have this exact same file in each of them ?

Reply:
Yes.. I could for now not think of a better way since the evoting feature needs some way to determine a subset type of what it receives?

Reply:
I guess it's ok for now, even if it's not ideal.